client/css/style.css

```diff
@@ -860,6 +860,10 @@ button,
 	color: #f00;
 }
 
+#chat .msg.toggle .time {
+	visibility: hidden;
+}
+
```

Comment:
> What do you mean?

Would need to do something special like setting time to `null`, and then doing if checks both on the server and the client so that it doesn't get reset or formatted. It's simply not worth doing that.